imports/components/graphs/lineGraph/index.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import {
+  VictoryChart,
+  VictoryAxis,
+  VictoryLine,
+  VictoryScatter,
+} from "victory";
+
+const styles = {
```

Comment:
@richarddubay I'd love to make the critical parts of this style object configurable via props:

`lineColor`
`dotColor` // for scatter
`tickStyles` // an object to override